src/content/admin/cleanup.mdx

```diff
@@ -117,3 +220,104 @@ In that case, the Garbage Collector will ignore only those specific resources wh
 You can set the `sleep` and `delete` periods in the [Admin Dashboard](#configuring-the-sleep-and-delete-periods) following the instructions above.
 
 If you are using Okteto Self-Hosted, you can also [configure this in the Helm Chart](self-hosted/helm-configuration.mdx#gc) along with options like a Slack webhook for notifications when resources are scaled to zero.
+
+## Troubleshooting
+
+### Common Issues and Solutions
+
+**Problem: Garbage Collector Not Sleeping Resources**
+
+*Symptoms:* Resources remain active despite exceeding the sleep period
+
+*Solutions:*
+1. **Check GC Status**: Go to **Admin → Garbage Collector** and verify it's enabled
+2. **Verify Timing**: Confirm the sleep period configuration and when the resource was last active
+3. **Check for Activity**: Look for recent deployments, `okteto up` sessions, or kubectl operations
+4. **Review Labels**: Ensure resources don't have `dev.okteto.com/persistent` label
+5. **Check Logs**: Look in **Admin → System Logs** for GC errors or warnings
+
+**Problem: Resources Deleted Unexpectedly**
+
+*Symptoms:* Important resources were deleted by GC
+
+*Solutions:*
+1. **Mark as Persistent**: Add `dev.okteto.com/persistent` label to critical namespaces
+2. **Adjust Periods**: Increase delete periods for your use case
+3. **Check Activity Tracking**: Ensure team knows what counts as "activity"
+4. **Recovery**: If recently deleted, check if backups are available
```

Comment:
For SaaS and BYOC, customers cannot do this, and I'm not sure our backup allows to restore specific/isolated namespaces